ui/src/message_popup/experience_page.jsx

```diff
@@ -237,102 +231,30 @@ export default React.createClass({
-              style={styles.button}
-              secondary={true}
-              label="Start" />
-          </div>
-        </div>
-      </VelocityTransitionGroup>
-    );
-  },
-
-  renderScaffoldingOptions() {
-    const {competencyGroupValue} = this.state;
+    const {isSolutionMode, name, competencyGroupValue, sessionLength, questions, shouldShowStudentCards, shouldShowSummary, helpType} = this.state;
     const competencyGroups = _.uniq(_.map(allQuestions, 'learningObjectiveId')).map((id) => {
       return _.find(learningObjectives, {id}).competencyGroup;
     });
     return (
-      <div>
-        <div style={styles.optionTitle}>Learning objectives to practice</div>
-        <RadioButtonGroup
-          name="competencyGroupValue"
-          valueSelected={competencyGroupValue}
-          onChange={this.onCompetencyGroupChanged}
-          style={_.merge({ padding: 20 }, styles.option)}>
-          <RadioButton
-            value={ALL_COMPETENCY_GROUPS}
-            label="All" />
-          {competencyGroups.map((competencyGroup) => {
-            return <RadioButton
-              key={competencyGroup}
-              value={competencyGroup}
-              label={competencyGroup} />;
-          })}
-        </RadioButtonGroup>
-        <Divider />
-        <div>
-          <div style={styles.optionTitle}>Session Length: {this.state.sessionLength} {this.state.sessionLength===1 ? "question" : "questions"}</div>
-          <Slider key={competencyGroupValue} value={this.state.sessionLength} min={1} max={this.state.questions.length} step={1} onChange={this.onSliderChange}/>
-        </div>
-        <Divider />
-        <div style={styles.optionTitle}>Scaffolding</div>
-        <div style={_.merge({ padding: 20 }, styles.option)}>
-          <Toggle
-            label="With student cards"
-            labelPosition="right"
-            toggled={this.state.shouldShowStudentCards}
-            onToggle={this.onStudentCardsToggled} />
-          <Toggle
-            label="Show summary after each question"
-            labelPosition="right"
-            toggled={this.state.shouldShowSummary}
-            onToggle={this.onSummaryToggled}/>
-          <div style={{margin: 10}}><Divider /></div>
-          <RadioButtonGroup name="helpOptions" valueSelected={this.state.helpType} onChange={this.onHelpToggled}>
-            <RadioButton
-              value="feedback"
-              label="With feedback and revision"
-              style={styles.radioButton}
-              />
-            <RadioButton
-              value="hints"
-              label="With hints available"
-              style={styles.radioButton}
-              />
-            <RadioButton
-              value="none"
-              label="With no help available"
-              style={styles.radioButton}
-              />
-          </RadioButtonGroup>
-        </div>
-        <Divider />
-      </div>
-    );
+      <InstructionsCard 
+        isSolutionMode={isSolutionMode}
```

Comment:
@david-rosales This is a great first step of refactoring, to pull this out and simplify the experience page!  There's a lot of parameters being passed in, so another step of simplification could be to not have the instructions/configuration pass all of the state back up (eg., maybe it's just "config" and "questions").